### PR TITLE
document default LEAD columns

### PR DESCRIPTION
**Pull request by fivetran-jamie**
## PR Overview
**This PR will address the following Issue/Feature:**
https://github.com/fivetran/dbt_marketo_source/pull/35

**This PR will result in the following new package version:**

v0.11.0 -- we are aliasing fields that existed but were not aliased before

**Please detail what change(s) this PR introduces and any additional information that should be known during the review of this PR:**

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt compile
- [x] dbt run –full-refresh
- [x] dbt run
- [x] dbt test
- [ ] dbt run –vars (if applicable)

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked and tagged
- [x] You are assigned to the corresponding issue and this PR
- [x] BuildKite integration tests are passing

### Detailed Validation
Please acknowledge that the following validation checks have been performed prior to marking this PR as "ready for review":
- [x] You have validated these changes and assure this PR will address the respective Issue/Feature.
- [x] You are reasonably confident these changes will not impact any other components of this package or any dependent packages.
- [x] You have provided details below around the validation steps performed to gain confidence in these changes.

see hex linked in height

### Standard Updates
Please acknowledge that your PR contains the following standard updates:
- Package versioning has been appropriately indexed in the following locations:
    - [x] indexed within dbt_project.yml
    - [x] indexed within integration_tests/dbt_project.yml
- [x] CHANGELOG has individual entries for each respective change in this PR
    
- [x] README updates have been applied (if applicable)
    
- [ ] DECISIONLOG updates have been updated (if applicable)
- [x] Appropriate yml documentation has been added (if applicable)

### dbt Docs
Please acknowledge that after the above were all completed the below were applied to your branch:
- [x] docs were regenerated (unless this PR does not include any code or yml updates)

### If you had to summarize this PR in an emoji, which would it be?
 
🍳

### CODE REVIEW

File: packages.yml [hunk 1]
```diff
@@ -1,3 +1,7 @@
 packages:
-- package: fivetran/marketo_source
-  version: [">=0.10.0", "<0.11.0"]
+# - package: fivetran/marketo_source
+#   version: [">=0.11.0", "<0.12.0"]
+# - local: ../../Marketing/Marketo/dbt_marketo_source
+- git: https://github.com/fivetran/dbt_marketo_source.git
+  revision: update/customer360
+  warn-unpinned: false
```
Comment (fivetran-joemarkiewicz) [inline packages.yml:7]: Reminder to swap before release

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,42 @@
+# dbt_marketo_source v0.11.0
```
Comment (fivetran-catfritz) [inline CHANGELOG.md:1]: typo 😺
```suggestion
# dbt_marketo v0.11.0
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,42 @@
...
+[PR #32](https://github.com/fivetran/dbt_marketo/pull/32) and Marketo Source [PR #35](https://github.com/fivetran/dbt_marketo_source/pull/35) include the following updates:
+
+## Feature Updates (includes potentially 🚨 breaking changes 🚨)
+- Ensures that `stg_marketo__lead` (and therefore `marketo__leads`) has and documents the below columns, all [standard](https://developers.marketo.com/rest-api/lead-database/fields/list-of-standard-fields/) fields from Marketo. Previously, peristed all fields found in your `LEAD` source table but only _ensured_ that the `id`, `created_at`, `updated_at`, `email`, `first_name`, `last_name`, and `_fivetran_synced` fields were included. If your `LEAD` table contains the following fields, nothing will change for you. If any of the following default columns are missing from your `LEAD` table, `stg_marketo__lead` will create a NULL version with the proper data type:
```
Comment (fivetran-catfritz) [inline CHANGELOG.md:5]: I'm wondering does this bit:
>   - `unsubscribed` -> aliased as `is_unsubscribed` (🚨 breaking change 🚨)
>   - `email_invalid` -> aliased as `is_email_invalid` (🚨 breaking change 🚨)

make this bit not be entirely true?:
> If your `LEAD` table contains the following fields, nothing will change for you.

Would their columns get renamed?

File: README.md [hunk 4]
```diff
@@ -89,7 +89,7 @@ vars:
     marketo__activity_delete_lead_enabled:  false     # Disable if you do not have the activity_delete_lead table 
 ```
 ## (Optional) Step 5: Additional configurations
-<details><summary>Expand for details</summary>
+<details open><summary>Expand/Collapse details</summary>
```
Comment (fivetran-catfritz) [inline README.md:92]: Ohh this is cool!

**Review by fivetran-catfritz — CHANGES_REQUESTED** [review]
One small change and a question!

**Review by fivetran-catfritz — APPROVED** [review]
lgtm for batching with the next update!

**Pull request merged**